src/main/java/grabber/sources/librarynovel_com.java

```diff
@@ -0,0 +1,105 @@
+package grabber.sources;
+
+import com.sun.javadoc.Doc;
+import grabber.Chapter;
+import grabber.GrabberUtils;
+import grabber.Novel;
+import grabber.NovelMetadata;
+import org.jsoup.Connection;
+import org.jsoup.HttpStatusException;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class librarynovel_com implements Source{
+    private final String name = "LibraryNovel";
+    private final String url = "https://librarynovel.com/";
+    private final boolean canHeadless = false;
+    private Novel novel;
+    private Document toc;
+
```

Comment:
Missing constructor methods:     
public librarynovel_com() {}

public librarynovel_com(Novel novel) {
    this.novel = novel;
}